protos/flyteidl/core/workflow.proto

```diff
@@ -205,3 +205,21 @@ message TaskNodeOverrides {
     // A customizable interface to convey resources requested for a task container. 
     Resources resources = 1;
 }
+
+// The workflow id is already used and the structure is different
+message WorkflowErrorExistsDifferentStructure {
+    string id = 1;
```

Comment:
let's use the workflow identifier, like here:  https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/workflow.proto#L16